composer.json

```
@@ -35,21 +35,20 @@
         "guzzlehttp/guzzle": "^7.2",
         "mockery/mockery": "^1.3.2",
         "monolog/monolog": "*",
-        "sensio/framework-extra-bundle": "^4.0 || ^5.5.1 || ^6.0",
         "doctrine/annotations": "^1.11",
-        "symfony/browser-kit": "^4.4 || ^5.0 || ^6.0",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/finder": "^4.4 || ^5.0 || ^6.0",
-        "symfony/phpunit-bridge": "v5.3.7",
-        "symfony/security-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/twig-bundle": "^4.4 || ^5.0 || ^6.0",
-        "twig/twig": "^2.13",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0",
-        "symfony/css-selector": "^4.4 || ^5.0 || ^6.0",
-        "symfony/expression-language": "^4.4 || ^5.0 || ^6.0",
+        "symfony/browser-kit": "^6.4 || ^7.0",
+        "symfony/console": "^6.4 || ^7.0",
+        "symfony/finder": "^6.4 || ^7.0",
+        "symfony/phpunit-bridge": "^7.0",
```

we could try forcing the latest patch version of the bridge, maybe that helps:

```suggestion
        "symfony/phpunit-bridge": "^7.0.1",
```